### DISCUSSION TITLE
Any plans on implementing grids from the osu! editor to osu!catch editor?

### DISCUSSION THREAD

**Discussion by snalgae** [ppy/osu#30712] [Ideas]
I think that organizing objects using patterns in osu!catch like any other gamemode is essential 
and the grid that was made for the osu! editor sounds like a tool that will be interesting to mess around with in the osu!catch editor. 

The osu!catch editor still lacks tools for structure and starting with something would be a good step forward.

**Comment by peppy** [comment]
Yes, I think this could be added.

**Reply by DizzyOracel** [reply]
saw a few people request this feauture again is it wip?

**Comment by akuza55555** [comment]
we need the grid
